webrecorder/browsertrix
# Update crawls list control bar UI

- Simplifies "Sort by" options (resolves https://github.com/webrecorder/browsertrix-cloud/issues/449)
- Adds crawl state (aka status) multi-select filter

### Manual testing
1. Run app and log in
2. Go to Crawls list view. Verify "View" filter defaults to "All Crawls" and "Sort by" defaults to "Date Created"
3. Select options from "View" filter. Verify crawls filter as expected
4. Sort crawls by dropdown options. Verify crawls are sorted as expected, and clicking the alternate sort icon with 2 arrows reverses the sort order.

### Screenshots
**Crawls list control bar:**
<img width="1118" alt="Screen Shot 2023-02-16 at 5 34 25 PM" src="https://user-images.githubusercontent.com/4672952/219512475-cc48d432-dfb8-4903-8c02-f6dd4a571577.png">

**Multiple status options selected:**
<img width="368" alt="Screen Shot 2023-02-16 at 5 35 47 PM" src="https://user-images.githubusercontent.com/4672952/219512521-f41cf77b-2860-4918-a530-3691f35edd25.png">

**Sort options:**
<img width="271" alt="Screen Shot 2023-02-16 at 5 36 35 PM" src="https://user-images.githubusercontent.com/4672952/219512594-3f2f2569-a8f3-4df9-bd98-27c3c354a49a.png">

**Empty state:**
<img width="1136" alt="Screen Shot 2023-02-16 at 5 35 58 PM" src="https://user-images.githubusercontent.com/4672952/219512624-d5fc3a79-e391-4adf-8a79-d853f8a005b0.png">

#### Differences from mockups:
- Added an empty state for when no crawls match -- could be improved & standardized later
- Using default Shoelace multi-select, which styles selected options into "tags" -- should be improved later
- No icons next to statuses, can be addressed with https://github.com/webrecorder/browsertrix-cloud/issues/608
